modules/terraform-cdp-azure-pre-reqs/modules/vnet/variables.tf

```diff
@@ -40,6 +40,20 @@ variable "vnet_cidr" {
 
 }
 
+variable "cdp_subnet_range" {
```

Comment:
As with AWS, to expose these variables to the user, they also need to be defined in the [modules/blob/feature/cdp_provider/modules/terraform-cdp-azure-pre-reqs/variables.tf](https://github.com/jimright/terraform-cdp-modules/blob/feature/cdp_provider/modules/terraform-cdp-azure-pre-reqs/variables.tf) files.

The value specified by the user, or the default value, is then passed vnet sub-module.

Reply:
Done!